docs/getting-started/browsers-devices.md

```diff
@@ -13,11 +13,11 @@ Bootstrap supports a wide variety of modern browsers and devices, and some older
 
 ## Supported browsers
 
-Bootstrap supports the **latest, stable releases** of all major browsers and platforms. On Windows, **we support Internet Explorer 9-11 / Microsoft Edge**. More specific support information is provided below.
+Bootstrap supports the **latest, stable releases** of all major browsers and platforms. On Windows, **we support Internet Explorer 9-11 / Microsoft Edge**. Alternative browsers which use the latest version of WebKit, Blink, or Gecko, whether directly or via the platform's web view API, are not explicitly supported – however, Bootstrap should (in most cases) display and function correctly in these browsers as well. More specific support information is provided below.
```

Comment:
Can you move the added text here to a new paragraph and break it into two separate sentences instead of using a dash there?